splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -1468,6 +1468,18 @@
                     "type": "string",
                     "maxLength": 100
                   },
+                  "inputHelperModule": {
+                    "type": "string",
+                    "maxLength": 100
+                  },
+                  "inputHelperModuleValidateInput": {
+                    "type": "string",
+                    "maxLength": 100
+                  },
+                  "inputHelperModuleStreamEvents": {
+                    "type": "string",
+                    "maxLength": 100
+                  },
```

Comment:
Let's remove it for now - let's assume that 1 helper = 1 modular input = 1 Splunk input.